### PR TITLE
forbid ntfs usage

### PR DESCRIPTION
**Pull request by sigmaSd**

### CODE REVIEW

File: lutris/util/system.py [hunk 1]
```diff
@@ -451,3 +451,21 @@ def is_running(process):
...
+    # import here to avoid cyclic import with lutris.gui.dialog
+    from lutris.gui.dialogs import ErrorDialog
+
+    def display_ntfs_warning():
+        ErrorDialog(
+            "<b>Using NTFS partition causes major issues!</b>\n")
+    out = None
+
+    if path:
+        out = subprocess.check_output(["df", "-Th", path]).decode("UTF-8")
```
Comment (PythonicChemist) [inline lutris/util/system.py]: Would this work instead: https://docs.python.org/3.7/library/os.html#os.fstatvfs ?

Comment (sigmaSd) [reply lutris/util/system.py]: I'm not sure how to use it the output is cryptic here is an example `os.statvfs_result(f_bsize=4096, f_frsize=4096, f_blocks=0, f_bfree=0, f_bavail=0, f_files=0, f_ffree=0, f_favail=0, f_flag=4106, f_namemax=255)
`

Comment (sigmaSd) [reply lutris/util/system.py]: weird it gives a lot of info about the file system but not what it is or I overlooked something

Comment (PythonicChemist) [reply lutris/util/system.py]: According to os.statvfs the support for filesystem ID field f_fsid was only added in Python 3.7.

Comment (sigmaSd) [reply lutris/util/system.py]: I compared os.statvfs out in ntfs and ext4 unfortunately nothing strikes out

Comment (sigmaSd) [reply lutris/util/system.py]: weird I have python 3.7.1 and I still don't see f_fsid in the output

Comment (sigmaSd) [reply lutris/util/system.py]: either way I guess that's too recent for what lutris targets

**Comment by sigmaSd** [comment]
btw @PythonicChemist I want to ask you this
```python
 if 'main_file' in self.game.config.game_config.keys():
```
is annoying me,I use it to avoid key not found error, would it be acceptable to wrap the next lines in try expect so I can remove this overhead

**Comment by sigmaSd** [comment]
@PythonicChemist  thanks a lot that's what I was looking for!
for the gui code, so you want to move ntfs check to gui dir? I just didn't find a suitable file for it there

**Comment by PythonicChemist** [comment]
No ntfs check is fine where it is. Just don't use gui code for it.

**Comment by sigmaSd** [comment]
Ah I get it I'll fix that^^

**Comment by strycore** [comment]
Please rebase your PR against the `next`  branch

**Pull request closed without merging**